base/setup/usetup/spapisup/cabinet.c

```diff
@@ -1147,32 +1180,6 @@ CabinetExtractFile(
             Status = CAB_STATUS_CANNOT_WRITE;
             goto UnmapDestFile;
         }
```

Comment:
You need also to move that ConvertDosDateTimeToFileTime call up as well, because it's the one that initializes the `FileTime` variable that is being used in the `memcpy(&FileBasic.LastAccessTime, &FileTime, sizeof(FILETIME));` call that has been moved.